modules/onboarding/organizational.tf

```diff
@@ -0,0 +1,11 @@
+# #---------------------------------------------------------------------------------------------
+# # Call Sysdig Backend to create organization with foundational onboarding
+# # (ensure it is called after all above cloud resources are created)
+# #---------------------------------------------------------------------------------------------
+# resource "sysdig_secure_organization" "oracle_organization" {
```

Comment:
I see this is commented out ?

Reply:
yes I think this should be uncommented?

Reply:
This was commented out because the `key` file issue. This was blocking the creation of an organization so commented for now while the fix was in place.